sphinx/ext/todo.py

```diff
@@ -209,6 +210,17 @@ def depart_todo_node(self, node):
     self.depart_admonition(node)
 
 
+def latex_visit_todo_node(self, node):
+    # type: (nodes.NodeVisitor, todo_node) -> None
+    title = node.pop(0).astext().translate(tex_escape_map)
+    self.body.append('\n\\begin{sphinxadmonition}{note}{%s:}' % title)
```

Comment:
py2 interpreter converts `str` object to `unicode` automatically if `title` is unicode object::
```
% python2
>>> type("str %s" % "str")
<type 'str'>
>>> type("str %s" % u"unicode")
<type 'unicode'>
```
So either is okay in this case.